esmvalcore/cmor/_fixes/CMIP6/BCC_ESM1.py

```diff
@@ -0,0 +1,66 @@
+# pylint: disable=invalid-name, no-self-use, too-few-public-methods
+"""Fixes for BCC-ESM1."""
+import numpy as np
+import iris
+
+from ..fix import Fix
+
+
+class allvars(Fix):
```

Comment:
This looks identical to the class defined in the file above, can you delete it from this file and just `from .BCC_CSM2_MR import allvars`?

Reply:
Changed as suggested. Will test this once the DKRZ machine is back on Monday.